### DISCUSSION TITLE
open .chm on a specific topic

### DISCUSSION THREAD

**Discussion by dannythevii** [sumatrapdfreader/sumatrapdf#3496] [Documentation]
is possible open chm on a specific topic via command line?
for use it in "hands made" programs

tanks
Daniele

**Comment by GitHubRulesOK** [comment]
I will be opening a bug report https://github.com/sumatrapdfreader/sumatrapdf/issues/3499 since there appears to be a current problem with opening names
but to answer your question use the page number for example

`"C:\Program Files\SumatraPDF\SumatraPDF.exe" -reuse-instance -page 52 "LJP2015_use_enww.chm"`

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4b34eea7-8b6b-4f0a-8a63-9b4252523866)
